generators/server/files.js

```diff
@@ -321,6 +318,16 @@ const serverFiles = {
                 }
             ]
         },
+        {
+            path: SERVER_MAIN_SRC_DIR,
+            templates: [
+                {
+                    condition: generator => !generator.skipClient,
+                    file: 'package/web/rest/SpaFilter.java',
```

Comment:
We should add a test case to ensure that the file is not generate when we choose no client framework.